### ISSUE TITLE
Add ability to run commands on different remotes

### ISSUE THREAD

**Issue by Dbz** [peterhurford/git-it-on.zsh#43]
It would be nice to be able to specify the remote on the commands. For example, I want to be able to run `gitit devops` to open the devops fork instead of the origin fork.

**Comment by peterhurford** [comment]
By fork, do you mean like `peterhurford/git-it-on.zsh` versus `devops/git-it-on.zsh`? Or do you mean branch?

**Comment by Dbz** [comment]
Yep, you got it. I mean fork.

**Comment by Dbz** [comment]
Hey @peterhurford are you willing to add this functionality?

**Comment by peterhurford** [comment]
Yep, I could maybe take a stab at it on Thursday after some deadlines. Or I'd be happy to review a PR.

I was thinking about the UI. Maybe `gitit --fork devops <args>`?

**Comment by Dbz** [comment]
Hey @peterhurford how about `gitit --remote devops <args>` ?
